jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -171,14 +172,16 @@ function set_timer_defaults {
     url=$(find_env "${prefix}_XA_CONNECTION_PROPERTY_URL" "${url}")
     if [[ $EJB_TIMER_DRIVER = *"mysql"* ]]; then
         if [ "x${url}" != "x" ]; then
-            EJB_TIMER_XA_CONNECTION_PROPERTY_URL="${url}?pinGlobalTxToPhysicalConnection=true"
+            EJB_TIMER_XA_CONNECTION_PROPERTY_URL="${url}?pinGlobalTxToPhysicalConnection=true\&amp;enabledTLSProtocols=${MYSQL_ENABLED_TLS_PROTOCOLS:-TLSv1.2}"
```

Comment:
Did we ever find out if not being able to include "pinGlobalTxToPhysicalConnection" in the mariadb connection url will cause us problems? Or perhaps we're waiting on QE?

Reply:
I didn't tested that, but apparently Mariadb does not requires it.

Reply:
conf applied to mariadb